python/nistoar/midas/dap/service/mds3.py

```diff
@@ -800,7 +800,7 @@ def _summarize(self, nerd: NERDResource):
             out["contactPoint"] = resmd["contactPoint"]
         if 'landingPage' in resmd:
             out["landingPage"] = resmd["landingPage"]
-        out["keyword"] = resmd.get("keyword", [])
+        out["keywords"] = resmd.get("keywords", [])
```

Comment:
In DAP records, the `data` property (being set via `out` in the above code) represents a _summary_ of the NERDm resource metadata (contained in `resmd` above).  We will not be changing the NERDm schema (standardized via a JSON-Schema definition)--only the how the selected information is provided in the summary.  Thus, the right-side reference to the property should remain singular (which I will fix in a subsequent commit).

Reply:
OK good to know, I thought those had to match